operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -157,14 +156,12 @@ func (c *CentralComponentSpec) GetAdminPasswordGenerationDisabled() bool {
 	return pointer.BoolPtrDerefOr(c.AdminPasswordGenerationDisabled, false)
 }
 
-// CentralDBEnabled returns true if central db should be created
-// TODO(ROX-13557): modify the logic to enable Postgres DB by default.
-func (c *CentralComponentSpec) CentralDBEnabled() bool {
-	if c == nil || c.DB == nil || c.DB.IsEnabled == nil {
+// IsExternalDB returns true if central DB is not managed by the Operator
+func (c *CentralComponentSpec) IsExternalDB() bool {
+	if c == nil || c.DB == nil {
```

Comment:
FWIW, `IsExternal` also checks for nil, so this whole function can probably be simplified as `return c != nil && c.DB.IsExternal()`?